server/middlewares/repaymentValidation.js

```diff
@@ -1,85 +1,85 @@
-import Repayments from '../models/repayment';
-import Loans from '../models/loan';
-import Helpers from '../utils/helpers';
-import db from '../models/db';
+// import Repayments from '../models/repayment';
```

Comment:
this is bad practice... You were being lazy... Please note this...Learning to make proper use of git would prevent this

Reply:
I apologise if I acted lazy and didnt didnt make proper use of git. I will always do better from now on.